MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1376,6 +1378,7 @@
 	_metalFeatures.maxTextureDimension = (16 * KIBI);
 	_metalFeatures.depthSampleCompare = true;
 	_metalFeatures.samplerMirrorClampToEdge = true;
+	_metalFeatures.clearColorFloatRounding = MVK_FLOAT_ROUNDING_DOWN;
```

Comment:
I actually don't recall seeing this problem with Intel GPUs. I think they use round-to-nearest as well.

Reply:
Thanks. I've just tested Intel and looks like you are right. 

I've changed the setting of `_metalFeatures.clearColorFloatRounding` to be driven by GPU `vendorID`.